core/client/client.go

```diff
@@ -333,9 +349,14 @@ func (c *Client) WaitForUserUpdate(ctx context.Context, m *entry.Mutation) (*ent
 	}
 }
 
+// WaitForRevision waits until a given map revision is available.
+func (c *Client) WaitForRevision(ctx context.Context, revision int64) error {
+	return c.WaitForSTHUpdate(ctx, revision+1)
```

Comment:
Are log and map revision numbers in sync? Possible confusion between comment about "map revision" and waiting for log STH. 

Also WaitForSTHUpdate comment says it waits for a tree size and this is passing a revision number.

Reply:
They are synced. I've added a method to more explicitly convert between revision and STH.TreeSize.